src/localization/en.json

```diff
@@ -120,12 +124,36 @@
     "editRequests": "Edit a vacation request",
     "edit": "Edit",
     "cancel": "Cancel",
-    "create": "Request vacation"
+    "create": "Request vacation",
+    "manageRequests": "Manage vacation requests"
   },
   "dataGrid": {
     "noRows": "No Rows"
   },
   "vacationsCard": {
-    "vacations": "Vacations"
+    "vacations": "Vacations",
+    "noUpcomingVacations": "No upcoming vacations",
+    "noUpcomingPendingVacations": "No upcoming pending vacations",
+    "upComingVacations": "You have {0} upcoming vacations",
+    "upComingPendingVacations": "You have {0} upcoming pending vacations",
+    "nextUpcomingVacation": "Next upcoming vacation",
+    "vacationType": "Vacation Type:",
+    "applicant": "Applicant:",
+    "timeOfVacation": "Time of Vacation:",
+    "status": "Tila:"
```

Comment:
Seems to be finnish word inside english localization.

Reply:
No voi paska. Korjataan.